pkg/init/Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS build
+# RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
```

Comment:
sorry, partly my fault. this line is not needed. It's only needed when you build a new root fs for a package. since we are not doing this in this stage it is not needed